src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/IThrottle.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace NuGet.Protocol
+{
+    public interface IThrottle
```

Comment:
Sorry missed that one  before. But still consider documenting this interface as it's not clear where and how to use it. Thanks!